src/main/java/org/takes/rq/RqLive.java

```diff
@@ -63,7 +63,7 @@ public RqLive(final InputStream input) throws IOException {
      */
     @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
     private static Request parse(final InputStream input) throws IOException {
-        final Collection<String> head = new LinkedList<String>();
+        final List<String> head = new LinkedList<String>();
```

Comment:
@lautarobock Why was this changed to `List`? I don't see any `List` specific operations.